EnumGenerator 버그

```
SpecsForEnum > [1] VALUE2, VALUE3, VALUE3, VALUE3, VALUE3, VALUE3, VALUE3, VALUE2, VALUE2, VALUE3, VALUE2, VALUE3, VALUE2, VALUE3, VALUE2, VALUE2, VALUE2, VALUE2, VALUE2, VALUE2 FAILED
    java.lang.AssertionError: 
    Expecting:
     <[VALUE3, VALUE2]>
    to contain:
     <[VALUE1, VALUE2, VALUE3]>
    but could not find:
     <[VALUE1]>
        at org.javaunit.autoparams.test.SpecsForEnum.sut_creates_arbitrary_enum_values(SpecsForEnum.java:56)
```

https://github.com/JavaUnit/AutoParams/pull/223/checks?check_run_id=2573716552

--- a/autoparams/src/test/java/org/javaunit/autoparams/test/SpecsForEnum.java
+++ b/autoparams/src/test/java/org/javaunit/autoparams/test/SpecsForEnum.java
@@ -53,7 +53,7 @@ class SpecsForEnum {
         set.add(value18);
         set.add(value19);
         set.add(value20);
-        assertThat(set).contains(EnumType.values());
+        assertThat(EnumType.values()).containsAll(set);
     }
 
 }

